games/devtest/mods/testnodes/drawtypes.lua

```diff
@@ -98,6 +98,16 @@ minetest.register_node("testnodes:allfaces", {
 	groups = { dig_immediate = 3 },
 })
 
+minetest.register_node("testnodes:allfaces_6", {
+	description = S("\"allfaces 6 Textures\" Drawtype Test Node").."\n"..
+		S("Transparent node with visible internal backfaces"),
+	drawtype = "allfaces",
+	paramtype = "light",
+	tiles = { "testnodes_allfaces.png^[colorize:red", "testnodes_allfaces.png^[colorize:orange", "testnodes_allfaces.png^[colorize:yellow", "testnodes_allfaces.png^[colorize:green", "testnodes_allfaces.png^[colorize:blue", "testnodes_allfaces.png^[colorize:purple" },
```

Comment:
Please break the line to max. 80 characters. https://dev.minetest.net/Lua_code_style_guidelines
or 95 if you apply the C++ guidelines.... https://dev.minetest.net/Code_style_guidelines#Don.27t_let_things_get_too_large

Reply:
Should be resolved.